Library/Homebrew/formulary.rb

```diff
@@ -257,6 +258,7 @@ def self.load_formula_from_api(name, flags:)
       resource "ruby-source" do
         tap_git_head = json_formula.fetch("tap_git_head", "HEAD")
         ruby_source_path = json_formula.fetch("ruby_source_path", "Formula/#{name}.rb")
+        ruby_source_path = CGI.escape(ruby_source_path)
```

Comment:
```suggestion
        ruby_source_path = URI.escape(ruby_source_path)
```
or, if it's in Ruby 2.6:
```suggestion
        ruby_source_path = URI::Parser.new.escape(ruby_source_path)
```